src/Agoda.Analyzers/RuleContent/AG0039MethodLineLengthAnalyzer.html

```diff
@@ -0,0 +1,69 @@
+﻿<p>
+    Long methods should be considered for refactoring
+</p>
+<ul>
+    <li>
+        This rule is to help people identify potentially long methods, so they can consider refactoring them.
+    </li>
+    <li>
+        The rule does not count lines of whitespace in the limit of 40 lines.
+    </li>
+    <li>
+        Try to break up long methods into smaller private methods that describe what's going on, however, don''t chain methods together to avoid long methods when refactoring.
+
+    </li>
+        <h2>Noncompliant Code Example</h2>
+        <pre>
+class MyClass()
+{
+    public void MethodThatDoesALotOfStuff()
+    {
+        return DoThisFirst();
+    }
```

Comment:
Not sure if this example of non-compliant code speaks for itself (as of why is it non-compliant).

Reply:
have made teh state that the code demostraights a separate line right before the example code so it's more obvious